torch/csrc/distributed/rpc/init.cpp

```diff
@@ -232,6 +232,9 @@ PyObject* rpc_init(PyObject* /* unused */) {
   // pythonRpcHandler is cleaned up in shutdown(), after
   // shutdown(), python objects returned from rpc python call can not be
   // resolved.
+  // TODO Once python object can be tagged as IValue and c10::ivalue::Future is
+  // implemented as generic Future<IValue>, we can consider all rpc call
+  // to return a future<IValue> later on.
   auto future = shared_ptr_class_<FutureMessage>(module, "Future")
```

Comment:
I am thinking about what Yanli is asking. 

This can be removed right, since "_invoke_rpc_torchscript" returns a `torch::jit::PythonFutureWrapper` now.